### DISCUSSION TITLE
Error with Make

### DISCUSSION THREAD

**Discussion by jfs5687** [idaholab/moose#28621] [Q&A Modules: Solid mechanics] [answered]
Hi @GiudGiud 

I am having an error while trying to use the _make_ command to compile MOOSE. The error that I am getting is:

Makefile:55: jfs5687/work/moose_directories/moose//framework/app.mk: No such file or directory
make: *** No rule to make target 'jfs5687/work/moose_directories/moose//framework/app.mk'.  Stop.

<img width="1021" alt="Screenshot 2024-09-13 at 10 39 41 PM" src="https://github.com/user-attachments/assets/27525f0c-3d40-4c1d-810f-d9c1103dee80">

I am unsure how to proceed. Any help is appreciated.

Thanks

**Comment by GiudGiud** [comment]
Hello

Is the moose folder under moose_directories full? Did it clone properly? Do you have access permissions to it?
It seems it's missing a file. Can you navigate there and find app.mk?

**Reply by jfs5687** [reply]
Yes, I can navigate to find app.mk in the moose folder within moose_directories. I believe that it cloned properly. The make command was running at one point with a different error, but it has since stopped and only given me the output I posted.

**Reply by GiudGiud** [reply]
weird. did you modify the Makefile? did you set MOOSE_DIR or another relevant environment vriable?

**Reply by jfs5687** [reply]
I did not modify the Makefile. I do not think that I set any environment variable.

**Reply by GiudGiud** [reply, marked as answer]
maybe save all your work with git commit in both myApp and moose, then use git clean -Xfd to clean those two repos and try a clean build
